gum/backend-darwin/gumobjcapiresolver.c

```diff
@@ -171,38 +171,58 @@ gum_objc_api_resolver_enumerate_matches (GumApiResolver * resolver,
   GumObjcApiResolver * self = GUM_OBJC_API_RESOLVER (resolver);
   GMatchInfo * query_info;
   gchar method_type;
-  GPatternSpec * class_spec, * method_spec;
+  GPatternSpec  * modifiers_spec, * class_spec, * method_spec;
```

Comment:
```suggestion
  GPatternSpec * class_spec, * method_spec, * modifiers_spec;
```

Let's assign variables (and thus declarations) in the same order as the groups.